client/my-sites/themes/themes-banner/style.scss

```diff
@@ -25,11 +18,22 @@
 		width: 100%;
 	}
 
+	a {
+		color: white !important;
```

Comment:
These importants are killing me. I know you moved them, but maybe we should remove them. Future cleanup PR?

Reply:
Removed! I'm not sure _why_ they were there, as the specificity is pretty good. We'll have to keep an eye out.